src/Distributions.jl

```diff
@@ -207,6 +209,7 @@ export
     sqmahal,            # squared Mahalanobis distance to Gaussian center
     sqmahal!,           # inplace evaluation of sqmahal
     location,           # get the location parameter
+    location!,          # provide storage for the location parameter (used in multivariate distribution mvlognormal)
```

Comment:
@kmsquire I opened an issue (https://github.com/JuliaStats/Distributions.jl/issues/459). The in-place variants would only be relevant to multivariate distributions, and the other multivariates that have been implemented to date may not have a location or scale, but I'll double-check it. 

I also realized that location and scale should be defined for some univariate distributions that use that terminology, yet some are missing. I opened https://github.com/JuliaStats/Distributions.jl/issues/460 for that.